src/screens/ChatScreen/SignedChatScreen.tsx

```diff
@@ -46,9 +39,12 @@ const SignedChatScreen = () => {
     (item: any) => item.user_id !== signedProfileId
   );
 
-  const renderChatItem = React.useCallback(({item, index}) => {
-    return <BaseChatItem type={SIGNED} item={item} index={index} />;
-  }, []);
+  const renderChatItem = React.useCallback(
+    ({item, index}) => {
+      return <BaseChatItem type={SIGNED} item={item} index={index} />;
+    },
+    [chats]
+  );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The dependency array of `useCallback` has been updated to include `chats`. This means that a new function will be created whenever `chats` changes. If `chats` changes frequently, this could lead to unnecessary re-renders and performance issues. Ensure that this change is necessary and beneficial.

```diff
- const renderChatItem = React.useCallback(({item, index}) => {
-   return <BaseChatItem type={SIGNED} item={item} index={index} />;
- }, []);
+ const renderChatItem = React.useCallback(
+   ({item, index}) => {
+     return <BaseChatItem type={SIGNED} item={item} index={index} />;
+   },
+   [chats]
+ );
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->